tooling/bundler/src/bundle/windows/msi/wix.rs

```diff
@@ -802,7 +802,14 @@ pub fn build_wix_app_installer(
       &msi_output_path,
     )?;
     rename(&msi_output_path, &msi_path)?;
-    try_sign(&msi_path, settings)?;
+
+    if settings.can_sign() {
+      try_sign(&msi_path, settings)?;
+    } else {
+      #[cfg(not(target_os = "windows"))]
+      log::warn!("Signing, by default, is only supported on Windows hosts, but you can specify a custom signing command in `bundler > windows > sign_command`, for now, skipping signing the installer...");
+    }
+
```

Comment:
wix is always windows
```suggestion
    }

```